openid-sharedsignals-framework-1_0.html

```diff
@@ -2378,43 +2413,34 @@ <h5 id="name-creating-a-stream">
+<ul class="normal">
+<li class="normal" id="section-7.1.1.1-3.1">
+                <code>events_requested</code><a href="#section-7.1.1.1-3.1" class="pilcrow">¶</a>
 </li>
-            </ul>
-<p id="section-7.1.1.1-5">delivery<a href="#section-7.1.1.1-5" class="pilcrow">¶</a></p>
-<ul class="normal ulEmpty">
-<li class="normal ulEmpty" id="section-7.1.1.1-6.1">
-                <p id="section-7.1.1.1-6.1.1"><strong>Receiver-Supplied</strong>, A JSON object containing a set of name/value pairs
-  specifying configuration parameters for the SET delivery method. The actual
-  delivery method is identified by the special key "method" with the value
-  being a URI as defined in <a href="#delivery-meta" class="auto internal xref">Section 11.2.1</a>.<a href="#section-7.1.1.1-6.1.1" class="pilcrow">¶</a></p>
+              <li class="normal" id="section-7.1.1.1-3.2">
+                <code>delivery</code> : Note that in the case of the POLL method, the <code>url</code> value is
+supplied by the Transmitter.<a href="#section-7.1.1.1-3.2" class="pilcrow">¶</a>
 </li>
-            </ul>
-<p id="section-7.1.1.1-7">format<a href="#section-7.1.1.1-7" class="pilcrow">¶</a></p>
-<ul class="normal ulEmpty">
-<li class="normal ulEmpty" id="section-7.1.1.1-8.1">
-                <p id="section-7.1.1.1-8.1.1"><strong>Receiver-Supplied</strong>, The Subject Identifier Format that the Receiver wants
-  for the events. If not set then the Transmitter might decide to use a type
-  that discloses more information than necessary.<a href="#section-7.1.1.1-8.1.1" class="pilcrow">¶</a></p>
+              <li class="normal" id="section-7.1.1.1-3.3">
+                <code>format</code><a href="#section-7.1.1.1-3.3" class="pilcrow">¶</a>
 </li>
             </ul>
-<p id="section-7.1.1.1-9">The following is a non-normative example request to create an Event Stream:<a href="#section-7.1.1.1-9" class="pilcrow">¶</a></p>
+<p id="section-7.1.1.1-4">If the request does not contain the <code>delivery</code> property, then the Transmitter
+MUST assume that the <code>method</code> is "urn:ietf:rfc:8936" (poll). The
+Transmitter MUST include a <code>delivery</code> property in the response with this
+<code>method</code> property and a <code>url</code> property.<a href="#section-7.1.1.1-4" class="pilcrow">¶</a></p>
+<p id="section-7.1.1.1-5">The following is a non-normative example request to create an Event Stream:<a href="#section-7.1.1.1-5" class="pilcrow">¶</a></p>
 <span id="name-example-create-event-stream"></span><div id="figcreatestreamreq">
 <figure id="figure-14">
-              <div class="lang-http sourcecode" id="section-7.1.1.1-10.1">
+              <div class="lang-http sourcecode" id="section-7.1.1.1-6.1">
 <pre>
 POST /ssf/stream HTTP/1.1
 Host: transmitter.example.com
 Authorization: Bearer eyJ0b2tlbiI6ImV4YW1wbGUifQo=
 
 {
   "delivery": {
-    "delivery_method": "urn:ietf:rfc:8935",
+    "method": "urn:ietf:rfc:8935",
```

Comment:
I looked at the already approved spec and it appears that the spec uses `delivery_method` key.
```
"delivery": {
    "delivery_method":
      "https://schemas.openid.net/secevent/risc/delivery-method/push",
      "url": "https://receiver.example.com/events"
  },
```
https://openid.net/specs/openid-sse-framework-1_0-ID1.html#rfc.section.7.1.2.2

Though I would have liked the `method` over `delivery_method` (as described earlier), that would break the backwards compatibility of transmitter's APIs.

Reply:
The currently approved spec uses `method` in the spec (see [sections 11.2.1.1. and 11.2.1.2](https://openid.net/specs/openid-sse-framework-1_0.html#rfc.section.11.2.1.1)), and mistakenly uses `delivery_method` in the examples.

Reply:
Thank you for clarifying it :)

Reply:
@FragLegs @tulshi noticed discrepancy around `endpoint_url` vs `url`. The spec mentions `endpoint_url` (see [sections 11.2.1.1. and 11.2.1.2](https://openid.net/specs/openid-sse-framework-1_0.html#rfc.section.11.2.1.1)) 

And examples show `url`. Could we align on this either in this PR or a brand new?